omod/src/main/webapp/fragments/field/locations.gsp

```diff
@@ -0,0 +1,22 @@
+<%
+    ui.includeJavascript("uicommons", "angular.js")
+    ui.includeJavascript("uicommons", "angular-common.js")
+    ui.includeJavascript("uicommons", "angular-resource.min.js")
+    ui.includeJavascript("uicommons", "angular-ui/ui-bootstrap-tpls-0.6.0.min.js")
+%>
+
+<div>
+    <div>
+        <p class="left">
+            <label>
+                Please select the default access location
```

Comment:
What are we using this for?

Reply:
I used this for the Patient Location Selection fragment Label. You can get the view from here, 

![image](https://user-images.githubusercontent.com/17599049/41507868-dbbe469c-7258-11e8-9b9b-d84d8de6af85.png)

Reply:
"Access location" do you mean the patient will access the system?

Reply:
And when you say "default" does this mean a patient is belonging to more than one location?

Reply:
Oh, So better to change this as the "Please select the location". It will be simple enough!